core/src/Cabana_Parallel.hpp

```diff
@@ -134,6 +133,17 @@ class TeamNeighborOpTag
 {
 };
 
+//! Angular neighbor operations are executed in serial on each neighbor thread.
+class SerialAngularNeighborOpTag
```

Comment:
I like this. We would need one more `VectorOpTag` (which implies, but isn't explicit about using team first neighbors) or `TeamVectorOpTag` for the fifth

Reply:
ok - so `VectorOp` tag would be needed for the 3rd level of parallelism in the second neighbor loop? If so, I would say we don't need to expose that yet. I'd rather do some research on performance and see if it is a good option in the first place. Per your comments, it may not vectorize well or could increase atomic operations.

Reply:
I would say that your point about just exposing the additional parallelism is sometimes useful (for 100 particles certainly, for 1000 slightly) on the GPU. I haven't seen a case where it degrades performance. 

And it still fits within your proposed change, with 2 tag args and 5 options